src/scaffolding/wizard/OpenStartPageStep.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { openStartPageAfterExtensionUpdate } from "../../commands/startPage/openStartPage";
+import { ScaffoldingWizardContext } from "./ScaffoldingWizardContext";
+
+export class OpenStartPageStep extends AzureWizardExecuteStep<ScaffoldingWizardContext> {
+    public constructor(public readonly priority: number) {
+        super();
+    }
+
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
I am not entirely sure what is the best practice for unused function arguments...

Reply:
If you don't use *any* of the arguments, you can simply omit them.  JS passes all arguments given it by the caller to the called function regardless of whether the function actually declares/uses them.